packages/theming/theme-tokens/src/highContrast/tokens-alias.ts

```diff
@@ -9,7 +9,7 @@ function processAliasTokens(aliasTokens: any) {
     for (const innerKey in aliasTokens[key]) {
       const entry = aliasTokens[key][innerKey];
       if (typeof entry === 'string' && entry.includes('PlatformColor')) {
-        const color = entry.substring(14, entry.length - 1);
+        const color = 'SystemColor' + entry.substring(14, entry.length - 1) + 'Color';
```

Comment:
Curious, what does system color mean here?

Reply:
It denotes that it's a brush "reserved"/built-in by the OS, as opposed to brushes that are user-defined.

The brush names for HC are documented here: https://docs.microsoft.com/en-us/windows/apps/design/accessibility/high-contrast-themes#contrast-colors

Reply:
AH, gotcha! macOS has a bunch of similar colors https://developer.apple.com/documentation/appkit/nscolor/ui_element_colors

Should the day come tokens have macOS PlatformColor's in them, I''' remember this is a place we can decode it!